source/grid/tria_accessor.cc

```diff
@@ -1082,6 +1082,102 @@ set (const internal::Triangulation::TriaObject<structdim> &object) const
 
 
 
+template <>
+TriaAccessor<0,1,1>
+TriaAccessor<1,1,1>::vertex_accessor (const unsigned int i) const
```

Comment:
You want me to do a partial specialization of the functions? I don't think it will work.